src/utils/string/index.ts

```diff
@@ -113,16 +114,20 @@ export const getDateCopy = (
   if (!time) {
     return "";
   }
-  const { tz, dateOnly, omitSeconds } = options || {};
+  const { tz, dateOnly, omitSeconds, omitTimezone } = options || {};
   let { dateFormat } = options || {};
   if (!dateFormat) {
     dateFormat = "MMM d, yyyy";
   }
   const finalDateFormat = dateOnly
     ? dateFormat
-    : `${dateFormat}, h:mm${omitSeconds ? "" : ":ss"} aa`;
+    : `${dateFormat}, h:mm${omitSeconds ? "" : ":ss"} aa${
+        omitTimezone ? "" : " O"
```

Comment:
```suggestion
        omitTimezone ? "" : " z"
```

This formats the timezones in the readable `EST`/ `UST` format.

Reply:
Wow seems like they need to update their docs! That's not what I expected from this...
<img width="698" alt="image" src="https://user-images.githubusercontent.com/9298431/233135787-541496dc-3691-47f8-b19c-a049eff2fab0.png">

Reply:
This function is using the format method from `date-fns-tz` not `date-fns` which includes support for readable timezones. https://github.com/marnusw/date-fns-tz#format